rust/extractor/src/main.rs

```diff
@@ -85,7 +87,7 @@ impl<'a> Extractor<'a> {
         }
         translator.emit_source_file(ast);
         translator.trap.commit().unwrap_or_else(|err| {
-            log::error!(
+            tracing::error!(
```

Comment:
very minor nit: for consistency with `info` and `warn` we should probably pull in `error` alongside them and just have
```suggestion
            error!(
```
here